repo/meta/schema/v3-repo-schema.json

```diff
@@ -23,6 +23,54 @@
       "pattern": "^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)$"
     },
 
+    "cliInfo": {
```

Comment:
please set indentation to 2 spaces to be consistent with the rest of the schema docs in the repo and the file.